### DISCUSSION TITLE
Self-service registration and login workflows

### DISCUSSION THREAD

**Discussion by rahul-muttineni-okcupid** [ory/kratos#2862] [Q&A]
Hello,

I want to be able to register/log in via social sign-in using an API flow since I want to perform this from Android/iOS and it's not clear how this works or whether this is even possible using the self-service registration/login API flows because according to the documentation, it is not currently supported. Does anyone know of alternative ways to accomplish this? Thanks!

**Comment by aeneasr** [comment]
Hi @rahul-muttineni-okcupid , this is on our roadmap! It would be great if we could get into a conversation into your use case and requirements. Please drop me a message at aeneas@ory.sh - thanks :)
